File: latihan3/java/ram.java
public class ram {
    // atribut
    private int capacity;   
    private int price;

    // constructor
    ram(){}

    public ram(int capacity, int price) {
        this.capacity = capacity;
        this.price = price;
    }

    // setter dan getter
    public int getCapacity() {
        return capacity;
    }

    public void setCapacity(int capacity) {
        this.capacity = capacity;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }
    
}

File: latihan3/php/ram.php
<?php
    class ram{
        // atribut
        private $price = 0;
        private $capacity = 0;
     
        // constructor
        function __construct($price=0, $capacity=0){
            $this->price = $price;
            $this->capacity = $capacity;
        }

        // Setter dan Getter

        function setPrice($price){
            $this->price = $price;
        }
        function getPrice(){
            return $this->price;
        }

        function setCapacity($capacity){
            $this->capacity = $capacity;
        }
        function getCapacity(){
            return $this->capacity;
        }

        // destructor
        function __destruct(){}

    }
?>